tests/config.example.yaml

```diff
@@ -0,0 +1,32 @@
+
+################################################################################
+##                             Network interface                              ##
+################################################################################
+server:
+  # Address to which we bind
+  address: ""      # default value
+  # Port on which we bind
+  port:    3836    # default value
+
+  # Endpoint of Agora
+  agora_endpoint: http://127.0.0.1:2826
+
+  # White IP list for IP-filtering ('\block_externalized', '\preimage_received')
+  white_ip_list:
+    - 127.0.0.1
+    - 127.17.0.0/16
+
+################################################################################
+##                               Database options                             ##
+################################################################################
+database:
+  filename: APP_ROOT/stoa/data/database
+
+################################################################################
+##                               Logging options                              ##
+################################################################################
+logging:
+  # The name of the folder where the logs are stored.
+  folder: APP_ROOT/stoa/logs/
```

Comment:
Instead of building a templating language here, it would be better to just make it relative to the working directory.

Reply:
Okay. I'll try to fix it.

Reply:
@Geod24 I fixed it according to your review.